### ISSUE TITLE
Live Lambda broken with `ws.Server is not a constructor`

### ISSUE THREAD

**Issue by SpencerDuball** [serverless-stack/serverless-stack#1725]
## Issue
I have run into an issue where the Live Lambda Dev will not start due to a websockets issue, `ws.Server is not a constructor`.

## Expected
The Live Lamda Dev should start up without an issue.

## Investigation
After doing some looking, I found this issue that references the same error https://github.com/websockets/ws/issues/1538#issuecomment-944859160. This appears to be an issue with `ws@8` when using ESM.

After looking further, I see that a `ws@^7.4.0` is marked in the [packages/cli/package.json](https://github.com/serverless-stack/serverless-stack/blob/e170ea3fb586cbe36e11beb8f9d9af4f420c2d7e/packages/cli/package.json) but not in the [packages/core/package.json](https://github.com/serverless-stack/serverless-stack/blob/e170ea3fb586cbe36e11beb8f9d9af4f420c2d7e/packages/core/package.json).

I believe this is where we are running into the issue as `ws` does not appear to have been directly added to the `core` package, however this is used in the following `core` file for local development: [packages/core/src/local/server.ts](https://github.com/serverless-stack/serverless-stack/blob/e170ea3fb586cbe36e11beb8f9d9af4f420c2d7e/packages/core/src/local/server.ts).

## Workaround
I tried installing `ws@7.4.0` into my backend package and got the Live Lambda Dev environment to work for anyone else stuck on this issue:
__Steps:__
1. `yarn workspace backend add ws@7.4.0` - Your workspace name might not match exactly, it might look like "@spencerduball/backend" but with your name of course.
2. `yarn sst start`
3. Now work locally on your lambdas!

# Error Info
## Package.json
All I did was start with a blank project, no modifications other than installing serverless stack had been made to this point.
```json
{
  "name": "spencerduball.com",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "start": "sst start",
    "build": "sst build",
    "deploy": "sst deploy",
    "remove": "sst remove",
    "console": "sst console",
    "typecheck": "tsc --noEmit",
    "test": "vitest run"
  },
  "devDependencies": {
    "@serverless-stack/cli": "^1.2.1",
    "@serverless-stack/resources": "^1.2.1",
    "@tsconfig/node16": "^1.0.2",
    "typescript": "^4.6.4",
    "vitest": "^0.12.9"
  },
  "workspaces": [
    "backend",
    "web"
  ]
}
```

## Error in Console
```sh
➜  spencerduball.com yarn sst start
Using stage: spencer-duball
Preparing your SST app

=======================
 Deploying debug stack
=======================


Deploying stacks

 ✅  spencer-duball-spencerduball-debug-stack (no changes)


Stack spencer-duball-spencerduball-debug-stack
  Status: no changes
  Outputs:
    BucketArn: arn:aws:s3:::spencer-duball-spencerduball-debug-bucket83908e77-1w606mhu05v5n
    BucketName: spencer-duball-spencerduball-debug-bucket83908e77-1w606mhu05v5n
    Endpoint: wss://5wn326qu53.execute-api.us-east-1.amazonaws.com/spencer-duball


===============
 Deploying app
===============


Deploying stacks

 ✅  spencer-duball-spencerduball-MyStack (no changes)


Stack spencer-duball-spencerduball-MyStack
  Status: no changes


==========================
 Starting Live Lambda Dev
==========================

ws.Server is not a constructor
```

**Comment by thdxr** [comment]
Hey Spencer can you try deleting your node_modules and reinstalling?

**Comment by SpencerDuball** [comment]
@thdxr I don't think it's a problem with my `node_modules/` I tried deleting and installing a few times last night when I encountered this issue to no avail. Just to make sure something wasn't wonky on my side I created a completely new project with `npm init sst`, ran `yarn install`, and then `yarn sst start` and ran into the same issue.

Let me know if there is any other information I can provide that would be helpful. Also since `ws` is being used in that `server.ts` file referenced above, is there a reason that `ws` is not included as a dev/regular dependency of the `core` package? I am thinking this still might be the issue but I am not an expert with monorepo structures to know the dependency hierarchy well enough to make a definitive statement.
